Sources/Shared/Data Manager/Model/Documents/File.m

```diff
@@ -74,8 +78,54 @@ - (void)dealloc {
     [_creator release];
     [_dateCreated release];
     [_dateModified release];
+    _naturalName = nil;
     [super dealloc];
 }
 
+-(NSString *) getNaturalName{
+    if (_naturalName!=nil) {
+        return _naturalName;
+    }
+    return _name;
+    
+}
+
+/*
+ In case of Group section, the name of the folder are genered automatically by the system & this names are not really nice (nature). The goal of this method is to convert this kind of name
```

Comment:
Write the comment so it's understood by Xcode : https://developer.apple.com/library/mac/documentation/DeveloperTools/Conceptual/HeaderDoc/tags/tags.html

Content:
Folders in the "Group" section are named automatically by the system & these names are not really nice.
The goal of this method is to convert this name into a more natural one without special characters.